blueocean-core-js/gulpfile.js

```diff
@@ -19,7 +19,7 @@ const fs = require('fs');
 // Options, src/dest folders, etc
 
 const config = {
-    clean: ["dist", "target"],
+    clean: ["dist"],
```

Comment:
Is there side effect of doing this? generated jar file has js code in it, can dist and target can ever get out of sync?

Reply:
I believe people do a clean anyway...